borsh/src/ser/mod.rs

```diff
@@ -637,3 +637,55 @@ impl<T: ?Sized> BorshSerialize for PhantomData<T> {
         Ok(())
     }
 }
+
+#[cfg(feature = "std")]
+impl<T> BorshSerialize for core::cell::Cell<T>
+where
+    T: BorshSerialize + Copy,
+{
+    fn serialize<W: Write>(&self, writer: &mut W) -> Result<()> {
+        <T as BorshSerialize>::serialize(&self.get(), writer)
+    }
+}
+
+#[cfg(feature = "std")]
+impl<T> BorshSerialize for core::cell::RefCell<T>
+where
+    T: BorshSerialize,
+{
+    fn serialize<W: Write>(&self, writer: &mut W) -> Result<()> {
+        <T as BorshSerialize>::serialize(core::ops::Deref::deref(&self.borrow()), writer)
+    }
```

Comment:
create a test for `try_borrow` error in `mod de_errors` module of the newly added `borsh/tests/test_cells.rs` file